libsrc/stdlib/outp.asm

```diff
@@ -8,11 +8,12 @@ EXTERN asm_outp
 .outp
 ._outp
 
-   pop af
+   pop bc
```

Comment:
Need to use `hl` here.
`bc` is used to hold the port.

Reply:
Oops.  Fixed.  Thanks.